### PR TITLE
add notes on granting scc access and clarify "project access"

### PR DESCRIPTION
**Pull request by pweil-**
Clarify how to grant access to a project since that question is often asked.  Add notes on nfs setup that you must grant the SCC in order to use it.

Fixes https://github.com/openshift/origin/issues/10660

@php-coder @adellape

### CODE REVIEW

File: admin_guide/manage_scc.adoc [hunk 1]
```diff
@@ -439,13 +439,18 @@ field of the SCCs.  Please see the
...
 
 Before adding an SCC to a user or group, you can first use the `scc-review`
 option to check if the user or group can create a pod. See the
 xref:../dev_guide/authorization.adoc#dev-guide-authorization[Authorization]
 topic for more information.
 
+Adding an SCC to a project is a question that is often asked.  SCCs are not granted
+to a project.  In this case you will be adding a service account to an SCC and either
+specifying a service account name on your pod or, when unspecified, running as the
+default service account.
```
Comment (php-coder) [inline admin_guide/manage_scc.adoc]: Let's emphasize `default` here because it's a name.

File: admin_guide/manage_scc.adoc [hunk 2]
```diff
@@ -459,6 +464,14 @@ $ oc adm policy add-scc-to-user <scc_name>  \
     system:serviceaccount:<serviceaccount_namespace>:<serviceaccount_name>
 ----
 
+[NOTE]
+====
+If you are currently in the project that the service account belongs to you may use the `-z` flag:
+----
+$ oc adm policy add-scc-to-user <scc_name> -z <serviceaccount_name>
```
Comment (php-coder) [inline admin_guide/manage_scc.adoc]: We can write the same but without using `NOTE`.

File: admin_guide/manage_scc.adoc (hunk 1 above)
Comment (pweil-) [inline admin_guide/manage_scc.adoc]: updated

File: admin_guide/manage_scc.adoc (hunk 2 above)
Comment (pweil-) [inline admin_guide/manage_scc.adoc]: updated

**Review by php-coder — APPROVED** [review]

**Comment by adellape** [comment]
Closing in favor of https://github.com/openshift/openshift-docs/pull/4664, PTAL there.

**Pull request closed without merging**